core-java/src/main/java/org/spine3/server/internal/CommandHandlerMethod.java

```diff
@@ -101,7 +103,8 @@ public static boolean isCommandHandler(Method method) {
         final Class<?> returnType = method.getReturnType();
         final boolean returnsMessageOrList =
                 Message.class.isAssignableFrom(returnType)
-                        || List.class.equals(returnType);
+                        || List.class.equals(returnType)
+                        || Void.TYPE.equals(returnType);
```

Comment:
Please have a separate class for ProcessManagerCommandHandler, which would allow void return type.